src/test/java/be/woutschoovaerts/mollie/ClientBuilderTest.java

```diff
@@ -39,9 +37,6 @@ void build_success_withProxy() {
                 .build();
 
         assertNotNull(client);
-        assertNotNull(Unirest.config().getProxy());
```

Comment:
This test was specifically for the proxy, so I think we still should check it of that it exists